web/locales/en.json

```diff
@@ -13,7 +13,10 @@
         "see": "see",
         "datasources": "data sources",
         "contribute": "Contribute by <a href=\"%s\" target=\"_blank\">adding your territory</a>",
-        "tip": "Tip: Click on a territory to start exploring"
+        "tip": "Tip: Click on a territory to start exploring",
+        "live": "Live",
+        "database": "Database",
```

Comment:
Actually the toolbar items shouldn't be translated as they are the name of our products.

Reply:
Ok, I'll rollback that edit